### DISCUSSION TITLE
where can i download `MobileNetV3Small`?

### DISCUSSION THREAD

**Discussion by MobilenetModel** [keras-team/keras#20075] [Q&A]
how to download models?

**Comment by MobilenetModel** [comment]
i tried searching for `mobilenet` at https://www.kaggle.com/search?q=mobilenet+in%3Amodels but i only saw `mobilenet_v1` and `mobilenet_v2` from `Tensorflow` :(

Where can I find and download MobileNetV3Small `from tensorflow.keras.applications`?

**Reply by abhaskumarsinha** [reply]
Hello there,

You might check very popular image library for Keras called `kim`: https://github.com/james77777778/keras-image-models#:~:text=MobileNetV2,MobileNetV3* . It offers support for `MobileNetV2` and `MobileNetV3` from @james77777778 .

Best Regards,
Abhas Kumar Sinha
